cmd/kubeflow-kernel/command/run.go

```diff
@@ -4,18 +4,22 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	restclientset "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/caicloud/ciao/pkg/backend"
 	kubeflowbackend "github.com/caicloud/ciao/pkg/backend/kubeflow"
 	"github.com/caicloud/ciao/pkg/config"
 	simpleinterpreter "github.com/caicloud/ciao/pkg/interpreter/simple"
 	"github.com/caicloud/ciao/pkg/kernel"
 	"github.com/caicloud/ciao/pkg/manager"
 	"github.com/caicloud/ciao/pkg/s2i"
+	configs2i "github.com/caicloud/ciao/pkg/s2i/configmap"
```

Comment:
Yeah, actually the name is a little confusing. Since the code is still in the early stage, and I do not figure out the final architecture, thus I think we could keep it. When we decided how to support image and volume and cm, we could refactor the interface here.